yarn.lock

```diff
@@ -372,6 +372,11 @@ safe-buffer@~5.2.0:
   resolved "https://registry.yarnpkg.com/safe-buffer/-/safe-buffer-5.2.0.tgz#b74daec49b1148f88c64b68d49b1e815c1f2f519"
   integrity sha512-fZEwUGbVl7kouZs1jCdMLdt95hdIv0ZeHg6L7qPeciMZhZ+/gdesW4wgTARkrFWEpspjEATAzUGPG8N2jJiwbg==
 
+semver@7.3.2:
+  version "7.3.2"
```

Comment:
We are mixing both npm and yarn as we have a yarn.lock file as well as package-lock files. I think we should decide on one tool.

Reply:
Good catch, VSCode is the only tool where we don't use yarn. I deleted the `yarn.lock` file and ran `npm install` to renew `package-lock.json` file ✅